docs/source/daal/algorithms/decision_forest/decision-forest-classification.rst

```diff
@@ -160,7 +160,8 @@ Examples
 
   .. tab:: Java*
 
-    .. note:: There is no support for Java on GPU.
+   .. note:: There is no support for Java on GPU.
```

Comment:
Maybe it is worth making this note just a sentence? It worked before, but now two notes are not the best way to present the info

Reply:
think worth deleting this altogether assuming deprecation